src/utils/queries.js

```diff
@@ -5,32 +5,50 @@ import {
   cartoNationsTable,
 } from "./config";
 
+/**
+ * Creates clause that only returns features in the provided latlng
+ */
+const containsClause = (bounds, and, geomField = "the_geom") =>
+  bounds
+    ? `
+${and ? "AND " : ""}ST_Contains(
+  ST_MakeEnvelope(${bounds[0][1]}, ${bounds[0][0]}, ${bounds[1][1]}, ${
+        bounds[1][0]
+      }, 4326),
+  ${geomField}
+)
+`
+    : "";
+
 /**
  * SQL queries that are passed to the CARTO SQL API
  * for more info on the SQL API see: https://carto.com/developers/sql-api/
+ *
+ * Bounds are optional
  */
-
-export const citiesCartoQuery = `
+export const citiesCartoQuery = (bounds) => `
 SELECT
   municipality, state, country, range, 
   policy_type, policy_summary, link, the_geom
 FROM ${cartoSheetSyncTable}
 WHERE the_geom is not null and admin_scale = 'City'
+  ${containsClause(bounds, true)}
```

Comment:
IMO this code would be a bit easier to follow if the `containsClause()` was only called if `bounds` is truthy when its parent query function is called. 

Also wonder about the argument for "AND"; would it make more sense to prepend "AND" before calling `containsClause` if `bounds` is truthy?

Instead of having a function argument for tacking on an "AND", you